recipes-cryptopuck/cryptopuck/cryptopuck_git.bb

```diff
@@ -15,7 +15,7 @@ RDEPENDS_${PN} = " \
 "
 
 SRC_URI = "\
-    git://git@github.com/platisd/cryptopuck;protocol=ssh;branch=master \
+    https://github.com/platisd/cryptopuck;protocol=ssh;branch=master \
```

Comment:
This doesn't work because `bitbake` doesn't understand this is a `git` repository (it's expecting a file, e.g. a `.zip`) and therefore doesn't know how to download and extract it.

The error I get is:

> ERROR: employee-getter-git-r0 do_fetch: No checksum specified for '/home/dimitris/dimitriOS/build/downloads/example-dimitriOS-cmake-project', please add at least one to the recipe:
> SRC_URI[sha256sum] = "5d81dc9648f77db5d0288129a6d016e297fd4f7a9edb35a841229d0d7b66c599"
ERROR: employee-getter-git-r0 do_fetch: Fetcher failure for URL: 'https://github.com/platisd/example-dimitriOS-cmake-project;protocol=ssh;branch=master'. Missing SRC_URI checksum

I think what you perhaps mean is for `ssh` to be changed to `https` in the `protocol` parameter, which I actually don't mind:

```
git://git@github.com/platisd/example-dimitriOS-cmake-project;protocol=https;branch=master \
```

Reply:
Ha, yes I just realized that and fixed it. I was waiting for the build to finish before pushing a fix. I'll do it now.